source/module_base/global_variable.h

```diff
@@ -282,6 +282,15 @@ extern std::string chg_extrap;
 extern int out_pot;
 
 extern std::string init_chg; //  output charge if out_chg > 0, and output every "out_chg" elec step.
+/// @brief method to initialize wavefunction
+/// @author kirk0830, 20230920
+extern std::string init_wfc; 
+/// @brief whether use the new psi initializer to initialize psi
+/// @author ykhuang, 20230920
+extern bool psi_initializer;
+/// @brief whether dump wavefunction after initialization, for debug
+/// @author ykhuang, 20230920
+extern bool wfc_dump;
```

Comment:
If you want to add some test parameters, please keep them in your module, such as psi_initializer.h